system-addon/test/unit/lib/TopSitesFeed.test.js

```diff
@@ -247,6 +247,19 @@ describe("Top Sites Feed", () => {
     });
   });
   describe("#refresh", () => {
+    it("should initialise _tippyTopProvider if it's not already initialised", async () => {
+      feed._tippyTopProvider.initialised = false;
```

Comment:
Should this be `initialised` or `initialized`?

Reply:
doh! nice catch @pdehaan 👍

Reply:
Ah thank you, English English getting me into trouble...